<launch>

  <rosparam command="load" file="/home/jiawei/Workspace/data/cams/tum/T_stereo.yaml" ns="so_dso"/>

  <node name="so_dso" pkg="direct_stereo_slam" type="so_dso_node" output="screen">
    <param name="cam0_topic" value="/cam0/image_raw" />
    <param name="cam1_topic" value="/cam1/image_raw" />
    <param name="calib0" value="/home/jiawei/Workspace/data/cams/tum/cam0/camera.txt" />
    <param name="gamma0" value="/home/jiawei/Workspace/data/cams/tum/cam0/pcalib.txt" />
    <param name="vignette0" value="/home/jiawei/Workspace/data/cams/tum/cam0/vignette.png" />
    <param name="calib1" value="/home/jiawei/Workspace/data/cams/tum/cam1/camera.txt" />
    <param name="gamma1" value="/home/jiawei/Workspace/data/cams/tum/cam1/pcalib.txt" />
    <param name="vignette1" value="/home/jiawei/Workspace/data/cams/tum/cam1/vignette.png" />
    <param name="mode" value="0" />
    <param name="preset" value="0" />
    <param name="scale_accept_th" value="15" />
  </node>

  </launch>
